src/color-picker/color-picker.scss

```
@@ -0,0 +1,181 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+@import '../button/borderless';
+
+$iui-color-field-size: $iui-baseline * 19;
+$iui-color-swatch-box-shadow: inset 0 0 0 $iui-xxs - 1 
+  rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+$iui-color-swatch-border-radius: 5px;
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  max-width: ($iui-l * 10) + ($iui-s * 9) + ($iui-sm * 2);
+  // max-width: (10 swatches per row) + (gaps) + (container padding)
+  max-height: ($iui-l * 10) + ($iui-s * 9) + ($iui-baseline * 2);
+  // max-height: (10 rows) + (gaps) + (container padding)
+  box-sizing: border-box;
+  border-radius: $iui-border-radius;
+  display: flex;
+  padding: $iui-baseline $iui-sm;
+  user-select: none;
+  overflow-y: auto;
+  @supports (overflow-y: overlay) {
+    overflow-y: overlay;
+  }
+
+  &.iui-advanced {
+    max-width: ($iui-color-field-size) + ($iui-sm) + ($iui-s) + ($iui-s) + ($iui-sm * 2);
+    // max-width: (color field width) + (color slider margins) + (color slider width) + (container padding);
+    flex-wrap: wrap;
+  }
+}
+
+@mixin iui-color-swatch {
+  height: $iui-l;
+  width: $iui-l;
+  cursor: pointer;
+  border-radius: $iui-color-swatch-border-radius;
+  background-color: var(--swatch-color);
+  margin-bottom: $iui-xs;
+  margin-right: $iui-xs;
+
+  @supports (gap: $iui-s) {
+    margin-bottom: 0;
+    margin-right: 0;
+  }
+
+  @include themed {
+    box-shadow: $iui-color-swatch-box-shadow;
+  }
+
+  &.iui-active {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1 t(iui-color-foreground-primary),
+        $iui-color-swatch-box-shadow;
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5)),
+        $iui-color-swatch-box-shadow;
+    }
+  }
+
+  &:focus {
+    outline: none;
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1
+        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5)),
```

I think this opacity might be too low, because it's hard to see focus effect in dark theme. Maybe try a higher value.
```suggestion
        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4)),
```

---

Changed opacity for color swatch focus